stdlib.go

```diff
@@ -135,16 +143,17 @@ const (
 )
 
 var (
-	logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeFileMsg = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeMsg     = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpMsg)
```

Comment:
As this is only used in a test, I think it can be moved there.

Reply:
I can be used for QoL reasons, like `logkit.NewStdlibAdapter(logger, Regexp(logkit.StdlibRegexpFull))`